ui/shared/tabs/failureSummary/FailureSummaryTab.jsx

```diff
@@ -163,26 +164,21 @@ class FailureSummaryTab extends React.Component {
     const jobLogsAllParsed =
       logs.length > 0 && logs.every((jlu) => jlu.parse_status !== 'pending');
 
-    let suggestionCounter = 0;
-    selectedJob.newFailure = false;
+    selectedJob.newFailure = 0;
     suggestions.forEach((suggestion) => {
-      suggestionCounter++;
-      // this allows us to focus on the top line
-      if (suggestionCounter < 2) {
-        suggestion.showNewButton = false;
-        // small hack here to use counter==0 and try for display only
-        if (
-          suggestion.search.split(' | ').length === 3 &&
-          (suggestion.failure_new_in_rev === true ||
-            (suggestion.counter === 0 && repoName === 'try'))
-        ) {
-          suggestion.showNewButton = true;
-          selectedJob.newFailure = true;
-        }
+      suggestion.showNewButton = false;
+      // small hack here to use counter==0 and try for display only
+      if (
+        suggestion.search.split(' | ').length === 3 &&
+        (suggestion.failure_new_in_rev === true ||
+          (suggestion.counter === 0 && repoName === 'try'))
+      ) {
+        if (selectedJob.newFailure === 0) suggestion.showNewButton = true;
```

Comment:
Can you try to split it across lines with curly braces, please? It's strange linting does not complain.